shared/getMetrics.js

```diff
@@ -1,4 +1,8 @@
-const convertAssertionPriority = require('./convertAssertionPriority');
+const convertAssertionPriority = priority => {
+    if (priority === 'REQUIRED') return 'MUST';
+    if (priority === 'OPTIONAL') return 'SHOULD';
+    return priority;
+};
```

Comment:
I noticed this function differs pretty significantly from the version currently on the server. Would you mind checking in with @howard-e about how to handle the discrepancy? I don't personally know how to address that.

Reply:
You are correct in that and this PR will need to account for that. This is now being handled through a dedicated utility file at [server/util/convertAssertionPriority](https://github.com/w3c/aria-at-app/blob/main/server/util/convertAssertionPriority.js).

I'm thinking this could be another candidate for the shared folder and the references to it from the server be updated for that.

Reply:
I added this to the shared section. Let me know your thoughts!